application.example.yaml

```diff
@@ -15,13 +16,14 @@ matrix.domain: ''
 # Signing keys #
 ################
 # Absolute path for the Identity Server signing key.
-# During testing, /var/tmp/mxisd.key is a possible value
 #
-# For production, use a stable location like:
+# For production, recommended location shall be one of the following:
 #   - /var/opt/mxisd/sign.key
 #   - /var/local/mxisd/sign.key
 #   - /var/lib/mxisd/sign.key
-key.path: ''
+#
+# The signing key is auto-generated during execution time if not present.
+key.path: '/var/tmp/mxisd.key'
```

Comment:
The value was left empty on purpose to force the user to choose the most appropriate path on the system.
If the key is lost or overwritten, there is no easy way to "cancel it", so it's crucial that it's the responsibility of the user.
I would keep it empty, unless you feel strongly about it and can tell why.

Reply:
Agreed with you, I'll reverse changes. In addition I suggest to write ```key.path: '/path/to/sign.key'``` to be coherent with line ```storage.provider.sqlite.database: '/path/to/mxisd.db'```

Reply:
According to our private discussion, I'll put back the empty value to ```key.path``` to avoid potential user problems.